dannce/interface.py

```diff
@@ -1649,8 +1597,7 @@ def build_model(params, netname, camnames):
                 params["chan_num"] + params["depth"],
                 params["n_channels_out"],
                 len(camnames[0]),
-                batch_norm=False,
-                instance_norm=True,
+                params["norm_method"],
```

Comment:
```suggestion
                norm_method=params["norm_method"],
```

@data-hound look good?

Reply:
Yes, this should solve the issue